internal/controller/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -673,6 +685,22 @@ filters:
     name: Cluster
     component: autocomplete
     hint: Specify a cluster ID or name.
+  - id: namespace
+    name: Namespace
+    component: autocomplete
+    autoCompleteAddsQuotes: true
+    category: targeteable
```

Comment:
what exactly does it mean to be targeteable in this context?
Also, searching the web, it seems we should remove the "e": "targetable".

Reply:
actually perhaps rename category as "endpoint" so it matches the terminology that we agreed on in the UI